CHANGELOG.md

```diff
@@ -7,7 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-## [2.12.1] - 2021-09-13 [DEPRECATED]
+### Added
+
+- I18n Ar.
+
+## [2.12.1] - 2021-09-13
```

Comment:
This changelog addition removed the `[DEPRECATED]` part. Could you change it to make it equal to the master version?